editor/edit-post/header/plugins/index.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * External dependencies
+ */
+import { map, isEmpty } from 'lodash';
+import { connect } from 'react-redux';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { NavigableMenu, withInstanceId, IconButton } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import { getEllipsisMenuItems } from '../../../api/ellipsis-menu';
+
+/**
+ * Renders a list of plugins that will activate different UI elements.
+ *
+ * @param   {Object} props The component props.
+ * @returns {Object} The rendered list of menu items.
+ */
+function Plugins( props ) {
+	const ellipsisMenuItems = getEllipsisMenuItems();
+
+	if ( isEmpty( ellipsisMenuItems ) ) {
+		return null;
+	}
+
+	const {
+		instanceId,
+	} = props;
+
+	/**
+	 * Handles the user clicking on one of the plugins in the menu
+	 *
+	 * Does nothing currently, but should be used to trigger the plugins sidebar
+	 *
+	 * @param {string} pluginId The plugin id.
+	 * @returns {void}
+	 */
+	function onSelect( pluginId ) {
+		props.onSelect();
+		ellipsisMenuItems[ pluginId ].callback();
+	}
+
+	const plugins = map( ellipsisMenuItems, ( menuItem ) => {
+		return {
+			value: menuItem.name,
+			label: menuItem.title,
+			icon: menuItem.icon,
+		};
+	} );
```

Comment:
I would've pointed out that this expr is equivalent to `pick( ellipsisMenuItems, [ 'value', 'label', 'icon' ] )`, but I don't think we need to construct this at all: since we never pass `plugins` (we only extract its properties into `IconButton`), we don't need to limit exposure. In other words, it's fine to drop `const plugins` and read directly from `ellipsisMenuItems` in the render map.